app/assets/src/components/common/Header.jsx

```diff
@@ -145,12 +154,22 @@ const UserMenuDropDown = ({
             target="_blank"
             rel="noopener noreferrer"
             href="https://assets.idseq.net/Privacy.pdf"
+            onClick={() =>
+              logAnalyticsEvent("Header_dropdown-privacy-policy-option_clicked")
+            }
```

Comment:
Do these onClick events consistently fire before the browser navigates to the new page? It's an edge case I'm not sure about.

Reply:
That's a good question. It was for sure a problem a number of years ago (10y?!) when I was doing something similar. However, in this case it is not an issue because of `target="_blank"`. In general, it should not much of an issue for IDSeq analytics because 1) we track pageviews as well when a page loads and we have the referrer for ordinary internal links 2) we don't have any external links that open in the same browser tab.

Reply:
Makes sense!